bc_obps/reporting/schema/report_operation.py

```diff
@@ -61,6 +62,7 @@ class ReportOperationIn(Schema):
     operation_name: str
     operation_type: str
     operation_bcghgid: str
+    registration_purpose: str
```

Comment:
Fields declared in the `Meta` class don't need to be re-specified manually in the schema itself, this is redundant